src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -93,7 +94,7 @@ public virtual IEnumerable<ICompletedFakeObjectCall> RecordedCallsInScope
             get { return FakeScope.Current.GetCallsWithinScope(this); }
         }
 
-        internal List<ICompletedFakeObjectCall> AllRecordedCalls
+        internal IEnumerable<ICompletedFakeObjectCall> AllRecordedCalls
         {
             get { return this.recordedCallsField; }

```

Comment:
> The call to ToList() is to prevent access to the backing field, ensuring that all mutation happens via RecordCall().

I'm not so sure it's a good idea. `ConcurrentQueue<T>.GetEnumerator()` already does a `ToList()`. So, when calling `GetRecordedCalls()`, this would cause 2 copies of the list to be created... isn't that overkill? If it was public, it might make sense (to prevent users from tampering with the FakeManager), but it's internal, so users can't access it anyway. Since the return type is `IEnumerable<T>`, we can't accidentally add something to the list (we would have to explicitly cast it, and we know we shouldn't do that).